lib/rules/no-irregular-whitespace/__tests__/index.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
```

Comment:
[suggestion] Perhaps, it seems we can reduce the duplication by exporting `IRREGULAR_WHITESPACES` as below:

```suggestion
const { messages, ruleName, IRREGULAR_WHITESPACES } = require('..');
```

Edit `lib/rules/no-irregular-whitespace/index.js`:

```js
// For test
module.exports.IRREGULAR_WHITESPACES = IRREGULAR_WHITESPACES;
```

But this change exports an essentially needless object. What do you think?

Reply:
I prefer deduplicating as well, but I haven't seen extra exports used in any other rule, so I wasn't sure if it is allowed to do in this environment.

Reply:
Make sense. It seems better to keep it as-is.